### PR TITLE
Small Updates

### PR DESCRIPTION
**Pull request by benwilson512**
Mostly just formatting and uncommenting those imports.

### CODE REVIEW

**Comment by benwilson512** [comment]
Notably, the last few commits you did fixed 134 test cases. We're down to just 360 failures!

**Comment by benwilson512** [comment]
This also now includes a PR from master that simplifies some of the runtime input code.

File: lib/absinthe/blueprint/input/value.ex [hunk 1]
```diff
@@ -27,7 +27,7 @@ defmodule Absinthe.Blueprint.Input.Value do
           | variable
 
   @type t :: %__MODULE__{
-          literal: literals | variable,
```
Comment (zacck-zz) [inline lib/absinthe/blueprint/input/value.ex:30]: Hmmm, do I understand this correctly? 
Here we are expanding the `literal` field in an input from a content field into a field containing `raw` struct that contains the literal and some meta about our input?

**Review by zacck-zz — COMMENTED** [review]
Seems like the main change here is the addition of the RawValue type

**Comment by zacck-zz** [comment]
Hmmm by my count  `64` test cases would be fixed with this merge

**Pull request merged**